### PR TITLE
Remove explicit answerIsCorrect variable, since this info is already captured in the feedback category.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: client/question/services/EventHandlerService.js [hunk 1]
```diff
@@ -153,7 +153,8 @@ tie.factory('EventHandlerService', [
        * @param {string} feedbackCategory The type of feedback shown to the
        *   user.
        * @param {string} code The user's submitted code.
-       * @param {boolean} success Whether or not the question was completed.
+       * @param {boolean} success Whether or not the entire question was
```
Comment (seanlip) [inline client/question/services/EventHandlerService.js]: @eyurko just to confirm, this is indeed supposed to be question, not task, right? It was actually sending task information before, so I just wanted to double-check what you had in mind.

Comment (eyurko) [reply client/question/services/EventHandlerService.js]: I thought it was supposed to be task information, but I don't really have a strong preference. Either way, we can just pull the last CodeSubmitEvent before a TaskCompleteEvent, so it's likely not _explicitly_ necessary.

Comment (seanlip) [reply client/question/services/EventHandlerService.js]: Hm, OK. I think question would be better than task, for two reasons:

- I don't think we are passing any info on which _task_ this corresponds to.
- If we wanted task completion, we can get it from the "successful" feedback category (but see the previous bullet point; we still don't know which task it corresponds to).

Alternatively, maybe we drop it altogether and rely on the other events (question start, question complete, task start, task complete) for progress.

I think I lean towards dropping it altogether so we don't have duplicate info. What do you think?

**Comment by eyurko** [comment]
I'm fine with that, though we should still have feedback category for
determining success

**Comment by seanlip** [comment]
Changes done, PTAL

File: client/question/services/EventHandlerService.js (hunk 1 above)
Comment (eyurko) [reply client/question/services/EventHandlerService.js]: SGTM

**Pull request merged**